haystack/utils/deepsetcloud.py

```diff
@@ -1204,6 +1204,44 @@ def update_eval_run(
         )
         return response.json()["data"]
 
+    def get_eval_run_results(
+        self, eval_run_name: str, workspace: Optional[str] = None, headers: dict = None
+    ) -> Dict[str, Any]:
+        """
+        Collects and returns the predictions of an evaluation run.
+
+        :param eval_run_name: The name of the evaluation run to update.
```

Comment:
The evaluation run to update or the evaluation run whose results we want to fetch?

Reply:
To fetch, copy-paste mistake. Thanks for catching this :sweat_smile: